### ISSUE TITLE
Cluster resources "not found" when running in a namespace

### ISSUE THREAD

**Issue by nolar** [zalando-incubator/kopf#249]
## Expected Behavior

Kopf watches over the cluster resources.

## Actual Behavior

Kopf cannot watch the cluster-scoped resources when running in a namespace.

It could do that when running on `pykube-ng`, as it was scanning and parsing the resources' scopes internally.

## Steps to Reproduce the Problem

```python
import kopf

@kopf.on.event('', 'v1', 'nodes')
def event_fn(spec, **kwargs):
    print(f"Node: {spec}")
```

Running with:

```
kopf run --verbose --namespace default handlers.py
```

## Specifications

```
kopf==0.23
```

**Comment by nolar** [comment]
Released as 0.23.1 hotfix — but kept open, as tests should be added (and maybe some cleanups).

**Comment by nolar** [comment]
Tests for discoveries are added in #252 for #251
